cmd/actuary/actuary.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+import (
+	"github.com/diogomonica/actuary/cmd/actuary/check"
+	"github.com/diogomonica/actuary/cmd/actuary/server"
+	"github.com/spf13/cobra"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/codes"
+	"os"
+)
+
+var (
+	mainCmd = &cobra.Command{
+		Use:           os.Args[0],
+		Short:         "Run actuary on a swarm",
+		SilenceUsage:  true,
+		SilenceErrors: true,
+	}
+)
+
+func init() {
+	mainCmd.AddCommand(
+		server.ServerCmd,
+		check.CheckCmd,
+	)
+}
+
+func main() {
+	//This seems problematic -- fix this
+	if c, err := mainCmd.ExecuteC(); err != nil {
+		c.Println("Error:", grpc.ErrorDesc(err))
```

Comment:
We should also probably not use `grpc.ErrorDesc(err)` before we check if it's an actual GRPC error